lms/djangoapps/instructor/views/api.py

```diff
@@ -2680,3 +2684,102 @@ def start_certificate_generation(request, course_id):
         'task_id': task.task_id
     }
     return JsonResponse(response_payload)
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_global_staff
+@require_POST
+def create_certificate_exception(request, course_id, white_list_student=None):
+    """
+    Add Students to certificate white list.
+    """
+    course_key = CourseKey.from_string(course_id)
+
+    try:
+        certificate_white_list = json.loads(request.body)
+    except ValueError:
+        return JsonResponse({
+            'success': False,
+            'message': _('Invalid Json data')
+        }, status=400)
+    try:
+        certificate_white_list, students = process_certificate_exceptions(certificate_white_list, course_key)
+    except ValueError as error:
+        return JsonResponse(
+            {'success': False, 'message': error.message, 'data': json.dumps(certificate_white_list)},
+            status=400
+        )
+
+    message = _('Students added to Certificate white list successfully')
```

Comment:
Also there are no other 'message' parameters being set -- if we are separating this out for translation purposes that is great, but we should include a translators' comment on the preceding line.  And as with response_payload, I would relocate this closer to the return.